src/variables.scss

```diff
@@ -1,7 +1,12 @@
+// Body
```

Comment:
question: I forgot to ask this in the last review, but... why SCSS variables? You know native CSS variables right? (eg `--light-text: #000`).

Nowdays, with those, there's no point in using SCSS variables. You just define them at `app.scss` and then it's available globally in all files. No need for `variables.module.scss` either.

Reply:
no reason in particular, I just thought I was using SCSS so => let's have SCSS variables 😅 it makes sense to move them to app.scss though given the user would set them, I would just provide some defaults somehow 🤔

Reply:
i think it would be something like this? --primary would be defined by the user and #4c21ea would be the fallback I define
```
$primary: var(--primary, #4c21ea);
```

Reply:
Exactly, yes, that's how you declare fallbacks!

P.S. SCSS in my opinion wins because of nesting. But even that is coming soon [Nesting in CSS](https://youtu.be/buChHSdsF9A?t=550)!